timber.php

```diff
@@ -319,6 +319,18 @@ public static function render( $filenames, $data = array(), $expires = false, $c
 		}
 		$output = self::compile( $filenames, $data, $expires, $cache_mode, true );
 		$output = apply_filters( 'timber_compile_result', $output );
+		return $output;
+	}
+
+	/**
+	 * @param array   $filenames
+	 * @param array   $data
+	 * @param bool    $expires
+	 * @param string  $cache_mode
+	 * @return bool|string
+	 */
+	public static function render( $filenames, $data = array(), $expires = false, $cache_mode = TimberLoader::CACHE_USE_DEFAULT ) {
+		$output = static::fetch( $filenames, $data = array(), $expires = false, $cache_mode = TimberLoader::CACHE_USE_DEFAULT );
```

Comment:
This line is incorrect, as it's rewriting the values of the variables with default values.

It should be 

    $output = static::fetch( $filenames, $data, $expires, $cache_mode);